ethcore/src/client/client.rs

```diff
@@ -1438,15 +1443,38 @@ impl RegistryInfo for Client {
 
 impl CallContract for Client {
 	fn call_contract(&self, block_id: BlockId, address: Address, data: Bytes) -> Result<Bytes, String> {
+		self.call_contract_from_sender(block_id, address, Address::default(), data)
+	}
+
+	fn call_contract_from_sender(&self, block_id: BlockId, address: Address, sender: Address, data: Bytes) -> Result<Bytes, String> {
```

Comment:
This function does a bit more than just call a contract with a sender now (since it also checks for `revert`). I think the function name should be changed to reflect this, or we just need to make sure that the docs on the function are clear about what's going on.

Reply:
Also on here, the signature is pretty long. Can you format it to fit in <100 characters?

Reply:
I added more explicit comments for `call_contract_from_sender` and `call_contract`. I'm actually not sure how to shorten the signature and save explicitness at the same time (if you take a look at input and output args their length is already 99 chars). Or did I understand this comment in a wrong way?